src/components/message-input/styles.scss

```diff
@@ -173,20 +173,38 @@
 
 .message-input {
   &__icons {
-    display: inline-flex;
-    justify-content: center;
+    display: flex;
     align-items: center;
-    padding-bottom: 2px;
-    margin-left: 10px;
+    gap: 8px;
 
     &-action {
       position: absolute;
-      right: 13px;
-      bottom: 9px;
+      right: 10px;
+      bottom: 8px;
       z-index: 1;
+
+      svg {
+        color: theme.$color-greyscale-11;
+      }
     }
   }
 
+  &__giphy-icon {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    height: 32px;
+    width: 32px;
+  }
+
+  &__record-voice-icon {
```

Comment:
This and the `giphy-icon` above could be under the same classname, however, the code and styles are a little hard to navigate so for ease of readability I have kept these separate for now, prior to further refactoring on a follow up task where this can be cleaned up.